README.md

```diff
@@ -144,3 +144,122 @@ assigned to the "AppSecurityGroup" so they can talk to the database as
 well as the load balancer, and receive SSH traffic from the bastion host.
 
 
+### Deploy
+
+New services are not always available in all AWS Regions when they are released. Please consult the
+[Region Table](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/) for
+more information.
+
+Deploy | Region Name | Region | VPC | Bastion | DB | Fargate | Elastic Beanstalk
+:---: | ------------ | ------------- | ------------- | ------------- | -------------  | ------------- | -------------
+[🚀 ][us-east-1-vpc] | US East (N. Virginia) | us-east-1 | 👍  ||||
+[🚀 ][us-east-1-vpc-bastion] | US East (N. Virginia) | us-east-1 | 👍  | 👍 |||
+[🚀 ][us-east-1-vpc-bastion-fargate] | US East (N. Virginia) | us-east-1 | 👍  | 👍  || 👍  ||
```

Comment:
I idly wonder if you could kill the App (Fargate) without database path as you do not provide this path for Elastic Beanstalk.

Reply:
The old EB template does not support creation without DBs, but I would like to change this in the future. Often, we see startups with existing DBs who want to migrate to the Startup Kit.

Reply:
:thumbsup: Seems very reasonable to support common migration use-cases.